data/bedrock/latest/types.yml

```diff
@@ -2043,6 +2055,24 @@ SoundType: varint =>
    - vault_deactivate
    - hurt_reduced
    - wind_charge_burst
+   - armor_crack_wolf
```

Comment:
there is an [empty value](https://github.com/pmmp/BedrockProtocol/blob/972373b6b8068963649f0a95163424eb5b645e29/src/types/LevelSoundEvent.php#L501) before this,
```
- wind_charge_burst
- _
- armor_crack_wolf